### DISCUSSION TITLE
Ternary Operation & Nullable

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#6224] [Language Ideas] [answered]
It would be really nice if the Ternary operator could work with nullable and non-nullable structs so that casting would not be necessary:

Ie.  Today:
```
            var Closed_At = Status != MatterImportJobStatus.Closed
                ? null
                : (DateTimeOffset?) Input.Attributes.Updated_At
                ;
```

Requested:
```
            var Closed_At = Status != MatterImportJobStatus.Closed
                ? null
                : Input.Attributes.Updated_At
                ;
```

**Comment by HaloFour** [comment, marked as answer]
See: https://github.com/dotnet/csharplang/blob/main/meetings/2020/LDM-2020-09-09.md#champion-nullable-enhanced-common-type

**Reply by svick** [reply]
Which makes this a duplicate of https://github.com/dotnet/csharplang/issues/33.
